infra/token-transfer-client/src/actions/otp.js

```diff
@@ -0,0 +1,79 @@
+import agent from '@/utils/agent'
+import { apiUrl } from '@/constants'
+
+export const VERIFY_OTP_PENDING = 'VERIFY_OTP_PENDING'
+export const VERIFY_OTP_SUCCESS = 'VERIFY_OTP_SUCCESS'
+export const VERIFY_OTP_ERROR = 'VERIFY_OTP_ERROR'
+export const SUBMIT_OTP_UPDATE_PENDING = 'SUBMIT_OTP_UPDATE_PENDING'
+export const SUBMIT_OTP_UPDATE_SUCCESS = 'SUBMIT_OTP_UPDATE_SUCCESS'
+export const SUBMIT_OTP_UPDATE_ERROR = 'SUBMIT_OTP_UPDATE_ERROR'
+
+function verifyOtpPending() {
+  return {
+    type: VERIFY_OTP_PENDING
+  }
+}
+
+function verifyOtpSuccess(payload) {
+  return {
+    type: VERIFY_OTP_SUCCESS,
+    payload
+  }
+}
+
+function verifyOtpError(error) {
+  return {
+    type: VERIFY_OTP_ERROR,
+    error
+  }
+}
+
+function submitOtpUpdatePending() {
+  return {
+    type: SUBMIT_OTP_UPDATE_PENDING
+  }
+}
+
+function submitOtpUpdateSuccess(payload) {
+  return {
+    type: SUBMIT_OTP_UPDATE_SUCCESS,
+    payload
```

Comment:
nit: `payload` param doesn't seem to be passed anywhere in the reducer for this action. Could remove it.

Reply:
👍